static/app/views/replays/deadRageClick/selectorTable.tsx

```diff
@@ -200,3 +218,14 @@ const StyledTextOverflow = styled(TextOverflow)`
 const StyledTooltip = styled(Tooltip)`
   display: inherit;
 `;
+
+const ProjectContainer = styled('div')`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: ${space(0.75)};
+`;
+
+const ProjectText = styled(TextOverflow)`
+  max-width: 100px;
+`;
```

Comment:
might not need this... the columns have a flexible width right?

I think we can set `width:100` in the BASE_COLUMNS array and it'll start out that size.

Reply:
okay so setting `widths` manually didn't seem to work, but i got rid of the `width` on `TextOverflow` bc that was definitely wrong. the table looks fine for now - only a little odd that there's so much spacing at first if you have a super long project name followed by super short project name, but not the worst. and you can always resize columns! so i think it's fine for now...

Reply:
<img width="739" alt="SCR-20231003-pglc" src="https://github.com/getsentry/sentry/assets/56095982/953d171e-4421-46b1-abb6-214ea6d5b73a">